bigtable/autoscaler/autoscaler.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample that demonstrates how to use Bigtable Stackdriver metrics to
+autoscale Google Cloud Bigtable."""
+
+import argparse
+import os
+import time
+
+from google.cloud import bigtable
+from google.cloud import monitoring
+
+import strategies
+
+CPU_METRIC = 'bigtable.googleapis.com/cluster/cpu_load'
+
+
+def get_cpu_load():
+    """Returns the most recent Bigtable CPU load measurement.
+
+    Returns:
+          float: The most recent Bigtable CPU usage metric
+    """
+    client = monitoring.Client()
+    query = client.query(CPU_METRIC, minutes=5)
+    return list(query)[0].points[0].value
+
+
+def scale_bigtable(bigtable_instance, up):
+    """Scales the number of Bigtable nodes up or down.
+
+    Args:
+           bigtable_instance (str): Cloud Bigtable instance id to scale
+           up (bool): If true, scale up, otherwise scale down
```

Comment:
I'd recommend `scale_up` as a more more descriptive name. Then, you can probably drop the args section and just describe the behavior in the description.